### DISCUSSION TITLE
Upload document to PDF file document loader via API

### DISCUSSION THREAD

**Discussion by garciagonpablo** [FlowiseAI/Flowise#1965] [Q&A]
Hey everyone! I been searching for a while if it is possible with Flowise to have an endpoint where I can upload a document in order to use it in the document loader to create embeddings from it. Is it possible? Will I have to create my own API to upload the document to my embeddings database? 
![image](https://github.com/FlowiseAI/Flowise/assets/123511393/9ae4f74f-9c07-45cf-8f4b-5558cf875f25)

**Comment by HenryHengZJ** [comment]
You can use the Vector Upsert API for that https://docs.flowiseai.com/using-flowise/api#vector-upsert-api
